test/langtools/tools/javac/reflect/CodeReflectionTester.java

```diff
@@ -63,8 +63,7 @@ static void error(String msg, Object... args) {
 
     static void check(Method method) throws ReflectiveOperationException {
         if (!method.isAnnotationPresent(CodeReflection.class)) return;
-        Field field = method.getDeclaringClass().getDeclaredField(method.getName() + "$op");
-        String found = canonicalizeModel(method, (String) field.get(null));
+        String found = canonicalizeModel(method, method.getCodeModel().orElseThrow());
```

Comment:
Was this even working after #162 ?

Reply:
Yes, i think we added this code early on before we added `Method::getCodeModel`